### PR TITLE
[libjpeg] requires debug postfix due to cmakes FindJPEG

### PR DESCRIPTION
**Pull request by Neumann-A**
#6656

also removed the renaming in the static case. I'll expect some regressions in projects not using cmake

### CODE REVIEW

**Comment by cenit** [comment]
keep the renaming, otherwise i think you are breaking QT and all related ports...

**Comment by Neumann-A** [comment]
6650 will also break Qt5 (zlib rename).

Can't wait until Qt switches to cmake.  

> Did you see any problem with it? Otherwise the good ol' rule of don't modify what's not broken is a golden rule :)

I like to minimize code and I will probably refactor the qt portfile a little bit. Too much copy and paste

**Comment by cenit** [comment]
> Can't wait until Qt switches to cmake.

november 2020.
It could be a blink or a slow agony, in the meantime...

**Comment by cenit** [comment]
Please study #5608 before touching symbols related to the path of the libraries. Much blood was wasted along the road before fixing those subtle problems

**Comment by Neumann-A** [comment]
@cenit I did not touch any how that. I just removed the 3 configure function calls and made them one.
hmm should probably use a find_library call to actually find the library paths and figure out if the windows build can also use full paths.

**Comment by cenit** [comment]
CURRENT_INSTALLED_DIR is now VCPKG_RELEASE_LIBDIR, that what I meant. If it works, sure. Just check it, I remember that it had been difficult to find the proper combination to build it and export it correctly (export in this case means producing a valid package recyclable as a nuget package for example on another computer....)

**Comment by NancyLi1013** [comment]
Here are some regressions from the current CI system:

|x64-windows-static|test|notes
|------------------|----|-----
|libfreenect2      |Fail|**Regression**
|butteraugli       |Fail|**Regression**

|arm64-windows|test|notes
|-------------|----|-----
|podofo       |Fail|**Regression**

[failureLogs.zip](https://github.com/microsoft/vcpkg/files/3230490/failureLogs.zip)

**Comment by JackBoosY** [comment]
Any progress?

File: ports/libjpeg-turbo/portfile.cmake [hunk 1]
```diff
@@ -34,6 +34,7 @@ vcpkg_configure_cmake(
     SOURCE_PATH ${SOURCE_PATH}
     PREFER_NINJA
     OPTIONS
+        -DCMAKE_DEBUG_POSTFIX=d
```
Comment (Neumann-A) [inline ports/libjpeg-turbo/portfile.cmake]: @JackBoosY According to the guidelines I am not allowed to do this. (It probably also interacts badly with configure scripts)
As such it is probably better to let libjpeg-turbo install a vcpkg-cmake-wrapper.cmake for FindJPEG which sets
`jpeg_names_debug` to the names observed in vcpkg before calling `find_package`.
Alternatively, we could set `JPEG_LIBRARY_DEBUG` before the `find_package` call

Comment (JackBoosY) [reply ports/libjpeg-turbo/portfile.cmake]: @Neumann-A Agreed.

**Comment by Neumann-A** [comment]
@JackBoosY I will close this since it requires another solution (see https://github.com/microsoft/vcpkg/pull/6658#discussion_r353601512)

**Pull request closed without merging**